### PR TITLE
scripts to download fortigate device schema

### PR DESCRIPTION
**Pull request by JieX19**
Scripts to download fortigate device schema from a server.

### CODE REVIEW

File: scripts/download_fgt_schema.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+echo "Enter the username:"
+read USER_NAME
+echo "Enter the password:"
+read PASSWORD
+
+SCHEMA_PATH=./fgt_schema.json
+
+curl -k -i -X POST https://${FGT_ADDRESS}/logincheck -d "username=${USER_NAME}&secretkey=${PASSWORD}" --dump-header /tmp/headers.txt
+
+curl -k -s --output ${SCHEMA_PATH} -X GET https://10.160.21.160/api/v2/cmdb?action=schema -b /tmp/headers.txt
```
Comment (chillancezen) [inline scripts/download_fgt_schema.sh:14]: hi Jie, 
the address here must not be hardcoded.
10.160.21.160 -> ${FGT_ADDRESS}

File: scripts/download_fgt_schema.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "Enter the fortigate address (e.g.: 192.168.102.32):"
+read FGT_ADDRESS
+echo "Enter the username:"
+read USER_NAME
+echo "Enter the password:"
+read PASSWORD
+
+SCHEMA_PATH=./fgt_schema.json
```
Comment (chillancezen) [inline scripts/download_fgt_schema.sh:10]: could we have a check for whether the schema file exists? if the schema file is present, let user choose whether to proceed.

File: scripts/download_fgt_schema.sh (hunk 1 above)
Comment (JieX19) [reply scripts/download_fgt_schema.sh:14]: Oops, I forgot to remove the hardcoded address. I am correcting it right now.  Both -k and --insecure enable us to use self-assigned certificate.

File: scripts/download_fgt_schema.sh (hunk 2 above)
Comment (JieX19) [reply scripts/download_fgt_schema.sh:10]: Thank you for your suggestion! it's more reasonable. I'll add the check logic right now.

**Pull request merged**